an error about import path of src/models/FullEvent.ts

```
node_modules/hltv/lib/models/FullEvent.d.ts:3:21 - error TS2307: Cannot find module 'enums/MapSlug'.

3 import MapSlug from 'enums/MapSlug';
                      ~~~~~~~~~~~~~~~

Found 1 error.
```
should be 
`import MapSlug from '../enums/MapSlug';`

--- a/src/models/FullEvent.ts
+++ b/src/models/FullEvent.ts
@@ -1,6 +1,6 @@
 import Team from './Team'
 import Event from './Event'
-import MapSlug from 'enums/MapSlug'
+import MapSlug from '../enums/MapSlug'
 import Country from './Country'
 
 export interface EventPrizeDistribution {

